CHANGELOG.md

```diff
@@ -19,6 +34,7 @@ If any files are returned, you should check to see if these files affected and a
 - Updated swagger documentation
 - Return 404 not found when calling file/dataset/space api endpoints with an invalid ID [#251](https://github.com/clowder-framework/clowder/issues/251)
 - Line breaks in welcome message breaks swagger build [#187](https://github.com/clowder-framework/clowder/issues/187)
+- Collections created using api route are now indexed upon creation. [#257](https://github.com/clowder-framework/clowder/issues/257)
```

Comment:
should these be here or part of 1.19?

Reply:
It looks like they should. Both PRs were merged into develop after the 1.18.1 - 2021-08-16 release. Fixing it now.

https://github.com/clowder-framework/clowder/pull/233
https://github.com/clowder-framework/clowder/pull/258